kerl

```diff
@@ -2526,20 +2560,36 @@ upgrade)
     fi
     ;;
 list)
-    if [ $# -ne 2 ]; then
-        list_usage
-        exit 1
-    fi
     case "$2" in
     releases)
+        if [ $# -ne 2 ] && [ $# -ne 3 ]; then
+            list_usage
+            exit 1
+        fi
+        if [ $# -eq 3 ] && [ "$3" != 'all' ]; then
+            list_usage
+            exit 1
+        fi
         check_releases
-        list_print "$2"
+        list_print "$2" "$3"
         l=t stderr "Run '$0 update releases' to update this list from erlang.org"
```

Comment:
Since we're touching this area of code, I think this message should not mention the erlang.org website any more (since it really parses tags from github by default)

```suggestion
        l=t stderr "Run '$0 update releases' to update this list"
```

Reply:
5ae4131